SketchUp/Sketchup/FrameChangeObserver.rb

```diff
@@ -61,33 +71,37 @@ class Sketchup::FrameChangeObserver
   #
   # @example
   #   class MyFrameChangeObserver
-  #     def frameChange(from_page, to_page, percent_done)
-  #       puts "From page #{from_page.name} to #{to_page.name} (#{percent_done * 100}%)"
+  #     def frameChange(from_scene, to_scene, percent_done)
+  #       if percent_done == 0.0
+  #         puts "Animating to scene '#{to_scene.name}':"
+  #       else
+  #         puts format("% 7.2f %",percent_done*100)
+  #       end
   #     end
   #   end
-  #   id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
+  #   @id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
   #
   # @note The +from_scene+ argument into this callback does not appear to
   #   be populated on the PC. You can store a variable that keeps track of
   #   the +to_scene+ and then use that on a subsequent Scene selection to
   #   determine the last Page that the user was on.
   #
-  # @param [Float] percent_done
-  #   The percentage of transition between
-  #   the two scene pages.
+  # @param from_scene [Sketchup::Page, nil]
```

Comment:
> That is also how the YARD docs presents thing. 

**Incorrect, Thomas.** 

YARD allows both ways. The documentation shows all the tags
 definition headings in **`@tag name description`** format.

But, not all of the examples throughout all of the manual have yet been updated to show this new ("name first") format.

Look at the YARD documentation specifically for:
The [`@attr`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attr) tag definition.
The [`@attr_reader`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attr_reader) tag definition.
The [`@attr_writer`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attr_writer) tag definition.
The [`@param`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#param) tag definition.
Look at the examples in the [`@overload`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#overload) tag.
And the examples for the [`@option`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option) tag.
The [`@see`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#see) tag definition.
The [`@yieldparam`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#yieldparam) tag definition.

**I don't understand why when we went through the Guidelines discussion, that you okayed using the new format (because it is better) and put the new format in the "[Documentation Guide.md](https://github.com/SketchUp/ruby-api-stubs/blob/master/Documentation%20Guide.md)" document, and now are reversing course !**